datalad/support/annexrepo.py

```diff
@@ -2328,7 +2328,16 @@ def _run_annex_command_json(self, command,
             else:
                 files = assure_list(files)
                 maxl = max(map(len, files))
-                chunk_size = CMD_MAX_ARG // maxl
+                chunk_size = max(
```

Comment:
I wonder why we only do this chunking at the `_run_annex_command_json` level.  Wouldn't the same limits apply to `_run_annex_command`?